iso690-full-note-cs.csl

```diff
@@ -233,21 +233,23 @@
   <macro name="issued">
     <choose>
       <if type="article-journal article-magazine article-newspaper" match="any">
-        <date variable="issued">
-          <date-part name="year" range-delimiter="&#160;&#8211;&#160;"/>
-        </date>
-        <choose>
-          <if variable="volume">
-            <text prefix=", " term="volume" form="short" suffix="&#160;"/>
+        <group delimiter=", ">
+          <date variable="issued">
+            <date-part name="year" range-delimiter="&#160;&#8211;&#160;"/>
+          </date>
+          <group delimiter="&#160;">
+            <text term="volume" form="short"/>
             <text variable="volume"/>
-          </if>
-        </choose>
-        <choose>
-          <if variable="issue">
-            <text prefix=", " term="issue" form="short" suffix="&#160;"/>
-            <text variable="issue"/>
-          </if>
-        </choose>
+          </group>
+          <choose>
+            <if variable="issue">
```

Comment:
@OTristanF, did you test my suggestion as-is? You shouldn't need these `<choose><if>` conditionals, as `<group/>` is an implicit conditional (see https://docs.citationstyles.org/en/stable/specification.html#group).